.github/workflows/push-tests.yml

```diff
@@ -75,7 +75,7 @@ jobs:
       - uses: actions/checkout@v2
       - name: Install dependencies
         run: |
-          npm ci
+          npm i
```

Comment:
@charles-testco since we don't save changes from CI/CD runs, what's the benefit of running `npm i` here? Thank you!

Reply:
Hi @tihuan we ran into an issue with dependencies and I was trying to force refresh of the dependencies and forgot to revert